modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase

```diff
@@ -1408,12 +1408,29 @@ definition {
 	}
 
 	@description = "LPS-139160 - Verify it is not possible to delete an Object via API after it was published"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CannotDeletePublishedObjectViaApi {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
 
-		// TODO LPS-141680 CannotDeletePublishedObjectViaApi pending implementation
+		ObjectAdmin.addObjectRelationshipViaAPI(
+			objectName_1 = "CustomObject",
+			objectName_2 = "CustomObject",
+			relationshipLabel = "Relationship",
+			relationshipName = "relationship",
+			relationshipType = "oneToMany");
```

Comment:
Hi @andreefarias
You don't need to create a relationship for this test. You need to create a field for that